src/Compiler/Checking/Expressions/CheckSequenceExpressions.fs

```diff
@@ -313,9 +319,15 @@ let TcSequenceExpression (cenv: TcFileState) env tpenv comp (overallTy: OverallT
             // Compile the pattern twice, once as a filter with all succeeding targets returning "1", and once as a proper catch block.
             let clauses, tpenv =
                 (tpenv, withList)
-                ||> List.mapFold (fun tpenv (SynMatchClause(pat, cond, innerComp, m, sp, _)) ->
+                ||> List.mapFold (fun tpenv (SynMatchClause(pat, cond, innerComp, m, sp, trivia)) ->
+                    let isTrueMatchClause =
```

Comment:
The logic is the same as cca 50 lines above. Could be calculated from SynMatchClause's trivia.

Reply:
Done